docs/reference/snapshot-restore/apis/snapshot-restore-apis.asciidoc

```diff
@@ -15,21 +15,22 @@ content may not be included yet.
 [discrete]
 [[snapshot-restore-repo-apis]]
 === Snapshot repository management APIs
-
-* <<clean-up-snapshot-repo-api,Clean up snapshot repository>>
-* <<delete-snapshot-repo-api,Delete snapshot repository>>
-* <<get-snapshot-repo-api,Get snapshot repository>>
 * <<put-snapshot-repo-api,Put snapshot repository>>
 * <<verify-snapshot-repo-api,Verify snapshot repository>>
+* <<get-snapshot-repo-api,Get snapshot repository>>
+* <<delete-snapshot-repo-api,Delete snapshot repository>>
+* <<clean-up-snapshot-repo-api,Clean up snapshot repository>>
```

Comment:
Not a blocker for this PR, but we should probably discuss whether we want APIs to be sorted alphabetically or not in the docs.

The order you have here makes sense to me, but it doesn't work for every API suite. You mind raising this in the next ES writers meeting?

Reply:
My thinking is that we order the APIs in the way that they are used. For example, *Delete* comes before *Put* alphabetically, but you cannot delete a snapshot or a repository if you haven't first used a put operation first. I'll make a note to discuss at the meeting 👍

Reply:
That makes sense for CRUD-only APIs, but some of our resources support more complicated operations. Indices are a good example:

https://www.elastic.co/guide/en/elasticsearch/reference/master/indices.html

That logic becomes harder to follow there.

I don't feel strongly either way, but we probably want to be consistent.